registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -447,6 +449,7 @@
     "sha256:925c528bbb4a90affbeee1dff51b614ba16df826ae9c8f9d4ddd675577410bef": ["v1.11.0-beta.0"]
     "sha256:d60be800331183b8bce700e7ea2871859554871a47dde7ee1a618fb8ce0045a7": ["v1.11.0-beta.1"]
     "sha256:dc86dc036cc67572aaaacec5e3ee74973c0dba906915a2790d03d483d02f9205": ["v1.11.0-beta.2"]
+    "sha256:b5ddc8ad1dbcba707761ddac484e582737d0ae236f359b92fc93e63502e4758b": ["v1.11.0-rc.0"]
```

Comment:
SHA `b5ddc8ad1dbc` matches kubeadm-bootstrap-controller v1.11.0-rc.0 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:b5ddc8ad1dbcba707761ddac484e582737d0ae236f359b92fc93e63502e4758b/details?inv=1&invt=Ab4ryw).